plugin/src/data/task.ts

```diff
@@ -6,7 +6,7 @@ import type { Priority, TaskId } from "@/api/domain/task";
 
 export type Task = {
   id: TaskId;
-  createdAt: string;
+  createdAt?: string;
```

Comment:
We are using the completed task's `completed_at` for this, so I think this doesn't need to be made optional